src/lib/serviceRoutines/postContextEntitiesByEntityId.cpp

```diff
@@ -51,12 +54,11 @@ std::string postContextEntitiesByEntityId
 )
 {
   std::string  entityId = (compV[0] == "v1")? compV[3] : compV[2];
```

Comment:
In contextBroker.cpp.
There are only two ways to enter this function, /v1/registry/XXX or /ngsi9/XXX.


But, what about "v2" ?
Someday we will have a version 2 of the protocol.
I also don't like this 'v1' very much. I actually almost changed it this morning ...

I would do "if (/ngsi9/XXX) entityId in compV[2]        else  in  compV[3]